src/Donations/resources/DonationDetails/app/components/PaymentInformation/Fields/HiddenFields/index.tsx

```diff
@@ -0,0 +1,16 @@
+import {useFormContext} from 'react-hook-form';
+
+/**
+ *
+ * @unreleased
+ */
+export default function HiddenFields() {
+    const {register} = useFormContext();
+    return (
+        <>
+            <input hidden {...register('amount')} />
+            <input hidden {...register('feeAmountRecovered')} />
+            <input hidden {...register('createdAt')} />
+        </>
+    );
+}
```

Comment:
I would just move these hidden fields to the container components so they're coupled together with their corresponding parts.

Reply:
Sounds good. I chose to place createdAt with the datepicker even though its split between that and the timepicker. Resolved: https://github.com/impress-org/givewp/pull/6752/commits/380558d201fc7ed7e7385c49d1ce15b93e1daf30